bitbots_vision/src/bitbots_vision/dynamic_color_space.py

```diff
@@ -69,25 +68,24 @@ def vision_config_callback(self, msg):
         :return: None
         """
         # Load dict from string in yaml-format in msg.data
-        vision_config = yaml.load(msg.data)
```

Comment:
yaml.load is deprecated

Reply:
For PyYAML-3.1 yaml.safe_load() is the safe method to load yaml.
Closed with ab393636e3e042fde61a03fd444f1b4133c182d6.